### PR TITLE
New icons for the Medical tools, defribillator, and reintrodution of the in-hand e-saw for traitors.

### PR DESCRIPTION
**Pull request by PestoVerde322**
## About The Pull Request

This PR introduces new icons for the medical tools, making it in line with the future material colour palette; alongside new icons for the medical table, defibrillator, portable defibrillator and defibrillator wall mounts.
On top of that this PR introduces a standalone e-saw for the traitor as a 5 tc item (same as the implant E-saw in matter of efficiency).

## Why It's Good For The Game

New icons for making the sever more distinguishable from the other and renewal of icons, on top of that, reintroduce a standalone non-implant e-saw for the traitor.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

(more screen will follow up later)

el tools

</details>

## Changelog
:cl:
add: the e-saw as a traitor item is back
imageadd: added new icons for all the medical tools
imageadd: added new icons for the defibrillator, wall mount and portable version
/:cl:

### CODE REVIEW

File: code/game/objects/items/melee/energy.dm [hunk 1]
```diff
@@ -175,14 +175,12 @@
...
-	icon_state = "saw"
-	icon_state_on = "saw"
+/obj/item/melee/transforming/energy/sword/esaw/implant //Energy Saw Arm Implant
+	icon_state = "implant-esaw_0"
+	icon_state_on = "implant-esaw_1"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	hitcost = 0
-*/
+
```
Comment (Sarchutar) [inline code/game/objects/items/melee/energy.dm:183]: why do you even need this second one when it's just the same as it's parent?

Comment (Sarchutar) [reply code/game/objects/items/melee/energy.dm:183]: okay, sorry misread this, disregard my comment

File: code/game/objects/items/melee/energy.dm [hunk 2]
```diff
@@ -175,14 +175,12 @@
...
 
-/*
-/obj/item/melee/transforming/energy/sword/cyborg/saw/implant //Energy Saw Arm Implant
-	icon_state = "saw"
-	icon_state_on = "saw"
+/obj/item/melee/transforming/energy/sword/esaw/implant //Energy Saw Arm Implant
+	icon_state = "implant-esaw_0"
+	icon_state_on = "implant-esaw_1"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
```
Comment (Sarchutar) [inline code/game/objects/items/melee/energy.dm:182]:
```suggestion
```
this part can go nonetheless since it really is the same as the parent one

**Comment by Pirill** [comment]
"I must feast on souls."

File: code/game/objects/items/melee/energy.dm (hunk 2 above)
Comment (PestoVerde322) [reply code/game/objects/items/melee/energy.dm:182]: as a matter of fact, I think i forgot the in-hand sprite for the Implant e-saw, will work on it tomorrow

**Comment by FriendlyContractor** [comment]
I feel like energy saw is not different enough from other meele weapons available to traitors to warrant its addition and would be more of unnecessary bloat than worthwhile addition . I would suggest making it more unique and distinct weapon. To give some ideas it could for example cause extra bleeding due to being super sharp or maybe have it deal bonus damage to limbs or even instant delimbing(probably would warrant upping a price slightly). Or maybe you could pull a special move during a surgery to violently kill a person by suprise

Sprites are all pretty good looking with the exception of shears which i think look kind of ugly. Anyway a great job!

**Comment by EvilDragonfiend** [comment]
bone setter? time to put this?
image

**Comment by Chirynnn** [comment]
Speedmerge Speedmerge Speedmerge Speedmerge Speedmerge Speedmerge Speedmerge Speedmerge Speedmerge

File: code/modules/uplink/uplink_items.dm [hunk 3]
```diff
@@ -2234,6 +2234,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	limited_stock = 2 //you can't use more than two!
 	restricted_roles = list("Shaft Miner")
 
+/datum/uplink_item/role_restricted/esaw
+	name = "Energy Saw"
+	desc = "A deadly energy saw inside your arm. Comes in a slick black finish."
```
Comment (PowerfulBacon) [inline code/modules/uplink/uplink_items.dm]: It's not exactly inside your arm

Comment (PestoVerde322) [reply code/modules/uplink/uplink_items.dm]: whoops

**Review by PowerfulBacon — APPROVED** [review]

**Comment by ghost** [comment]
Are some of the sprites ported? They seem extremely familiar, I'm just not sure where from.

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**